src/getSumWithClosure.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * Implement getSumWithClosure function:
+ *
+ * Function works with next pattern: getSumWithClosure(a)(b) === a + b. Double
+ * braces is not a mistake.
+ *
+ * getSumWithClosure(1)(2) === 3
+ * getSumWithClosure(1)(11) === 12
+ * getSumWithClosure(1)(-1) === 0
+ *
+ * @param {number} a
+ *
+ * @return {function}
+ */
+function getSumWithClosure(a) {
```

Comment:
name is kind of weird. maybe just sum(). And add hint in the docs to use closure?